crates/plugins/mediator-coordination/src/didcomm/bridge.rs

```diff
@@ -36,25 +37,37 @@ impl DIDResolver for LocalDIDResolver {
             return Ok(Some(self.diddoc.clone()));
         }
 
-        if !did.starts_with("did:key:") {
+        if did.starts_with("did:peer:") {
+            // Add logic to resolve Peer DIDs here
+            let method = DidPeer::new_with_format(PublicKeyFormat::Jwk); // Assuming a Peer DID resolver
+            match method.expand(did) {
+                Ok(diddoc) => Ok(Some(
+                    serde_json::from_value(json!(Document {
+                        service: Some(vec![]),
```

Comment:
`DidPeer` already contains a `service` field.

Reply:
Not always, and that's the problem. Because the type from the `didcomm` dependency requires the field, even if empty.

Reply:
We can pause this for later. I'm pretty sure we can do without serializing and deserializing. They cause a lot of CPU and memory overheads.
> Most serialization frameworks like [serde](https://serde.rs/) define an internal data model that consists of basic types such as primitives, strings, and byte arrays. This splits the work of serializing a type into two stages: the frontend and the backend. The frontend takes some type and breaks it down into the serializable types of the data model. The backend then takes the data model types and writes them using some data format such as JSON, Bincode, TOML, etc. This allows a clean separation between the serialization of a type and the data format it is written to.
>
> Serde describes [its data model](https://serde.rs/data-model.html) in the serde book. Everything serialized with serde eventually boils down to some combination of those types!
>
> A major downside of traditional serialization is that it takes a considerable amount of time to read, parse, and reconstruct types from their serialized values.
>
> In JSON for example, strings are encoded by surrounding the contents with double quotes and escaping invalid characters inside of them:
>
> { "line": "\"All's well that ends well\"" }
          ^^                          ^ ^
> numbers are turned into characters:
>
> { "pi": 3.1415926 }
        ^^^^^^^^^
> and even field names, which could be implicit in most cases, are turned into strings:
>
> { "message_size": 334 }
  ^^^^^^^^^^^^^^^
All those characters are not only taking up space, they're also taking up time. Every time we read and parse JSON, we're picking through those characters in order to figure out what the values are and reproduce them in memory. An f32 is only four bytes of memory, but it's encoded using nine bytes and we still have to turn those nine characters into the right f32!